client/app/pages/queries/VisualizationEmbed.jsx

```diff
@@ -120,8 +121,7 @@ function VisualizationEmbedFooter({
         <span className="hidden-print">
           <Tooltip title="Open in Redash">
             <Link.Button className="icon-button" href={queryUrl} target="_blank">
-              <i className="fa fa-external-link" aria-hidden="true" />
-              <span className="sr-only">Open in Redash</span>
+              <WithIcon alt="Open in Redash" />
```

Comment:
Those usages all look a little bit weird: you think that it's just some generic component that add (some) icon. However, it's hard to guess that by default it adds icon for external links and uses `alt` attribute to add some extra text. Currently I don't have any good suggestions; maybe, just rename this component to better reflect its purpose?